### PR TITLE
Add an indentation test

### PR DESCRIPTION
**Pull request by DavidSCN**

### CODE REVIEW

File: .clang-format [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #
-# The clang-format (Clang 6) style file used by deal.II.
+# The clang-format style file used by deal.II (Clang 11 instead of Clang 6)
```
Comment (MakisH) [inline .clang-format:2]: Just FYI, in preCICE we require specifically clang-format 8, so people will likely have this one installed: https://github.com/precice/precice/blob/develop/tools/formatting/format-all (I don't remember the reason)

**Review by MakisH — APPROVED** [review]
I guess that this will complain if the files are not formatted? I like this, let's add it! The files look reasonable, but maybe we need to decide whether we keep clang-format 11 or we use the (preferred by preCICE) 8.

File: .clang-format (hunk 1 above)
Comment (DavidSCN) [reply .clang-format:2]: All in all, I think there are not that big differences between different versions. On my old machine, I used Clang 6. Updating on my recent system to Clang 11 doesn't show any differences. If the version is not suitable, we can change it later on.

**Comment by DavidSCN** [comment]
> I guess that this will complain if the files are not formatted? I like this, let's add it! The files look reasonable, but maybe we need to decide whether we keep clang-format 11 or we use the (preferred by preCICE) 8.

I will test it now with a dummy commit. Let's go with Clang 11 (which is also the default on recent Ubuntu systems), In general, formatting the code with Clang 8 doesn't give any difference.

**Comment by DavidSCN** [comment]
Works nicely

**Comment by fsimonis** [comment]
@DavidSCN Please be aware that various versions of clang-format can (and often do) result in different outputs. This is why we require version 8 in precice.

**Pull request merged**